### PR TITLE
Update Program.cs from pr

### PR DESCRIPTION
**Pull request by jacwu**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -31,7 +31,7 @@ async static Task Main(string[] args)
                 {
                     if (e.Result.Reason == ResultReason.RecognizedSpeech)
                     {
-                        Console.WriteLine($"{e.Result.SpeakerId}:{e.Result.Text}");
+                        Console.WriteLine($"{e.Result.SpeakerId}: {e.Result.Text}");
                     }
```
Comment (jacwug) [inline Program.cs:35]: looks ok?

Comment (jacwu) [reply Program.cs:35]: yes, it is

**Review by jacwug — COMMENTED** [review]
looks something wrong?

**Comment by jacwug** [comment]
not good right now

**Pull request closed without merging**